.gitlab-ci.yml

```diff
@@ -21,9 +21,7 @@ Install dependencies and build assets:
   script:
     - COMPOSER_MEMORY_LIMIT=-1 composer install --no-scripts --no-progress
     - COMPOSER_MEMORY_LIMIT=-1 composer run-script post-autoload-dump
-    - volta install node
-    - npm ci --no-progress
-    - node_modules/.bin/encore production
+    - php bin/console sass:build
```

Comment:
Moet hier ook niet onderstaande boven:
```bash
- php bin/console importmap:install
- php bin/console  asset-map:compile 
```

Reply:
importmap:install toegevoegd 2114ce8 - env=prod is niet nodig denk ik, zie ook vvkp https://git.sumocoders.be/vvkp/platform-vvkp-be/-/jobs/327735